.github/workflows/CI.yml

```diff
@@ -41,6 +41,10 @@ jobs:
         run: ./bin/npm test
         # Q: Why are we using some random test section when the package.json has a test script?
         # A: So that we ensure we use the bundled version of node to run our tests
+      - if: failure()
+        name: Print Logs for exceptions
+        #run: echo "$(cat ./script/log.txt)" >> $GITHUB_OUTPUT
+        run: echo "$(cat ./script/log.txt)"
```

Comment:
I _think_ this should work across all platforms.

_Particularly this suggested snippet should make the step work as intended on Windows._ Due to running this bash syntax step on bash on all OSes, rather than it failing on Windows' default CMD.exe.

```suggestion
      - if: failure()
        name: Print Logs for exceptions
        run: if [[ -f "./script/log.txt" ]]; then cat ./script/log.txt; else echo "No log file found at ./script/log.txt"; fi
        shell: bash
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

This would also have the step always report "success" status, more than likely, regardless of whether there was a log file to print.

If having the "print logs" step show as failed is useful when there is no log file to print, then we can do `exit 1` after the `echo "no log file found..."` bit, to have it mark the step failed if the log file is missing.

As you mentioned above, since the overall run is already failed at this point, that would just be for informational purposes at a glance, to see at a glance whether there was any log to print or not.

Reply:
This change looks rad! I'll add it in